doc/requirements.txt

```diff
@@ -4,6 +4,7 @@ matplotlib
 matplotlib-inline
 mpmath
 myst-parser
+scipy
```

Comment:
Do we actually need scipy to build the docs? If not, we shouldn't put it here.

Reply:
As we discussed, yes we do, because we use scipy to create the plot.